template/Makefile.in

```
@@ -479,8 +478,9 @@ probes.stamp: $(DTRACE_REBUILD_OBJS)
 
 probes.$(OBJEXT): $(srcdir)/probes.d $(DTRACE_REBUILD:yes=probes.stamp)
 	@$(ECHO) processing probes in object files
-	$(Q) $(RM) $@
-	$(Q) $(DTRACE) -G -C $(INCFLAGS) -s $(srcdir)/probes.d -o $@ $(DTRACE_REBUILD_OBJS)
+	@# n.b. the dtrace script looks at the $CFLAGS environment variable to decide
+	@# how to assemble probes.o; so we need to actually _export_ $(CFLAGS)
+	$(Q) /usr/bin/env CFLAGS="$(CFLAGS) $(XCFLAGS) $(CPPFLAGS)" $(DTRACE) -G -C $(INCFLAGS) -s $(srcdir)/probes.d -o $@ $(DTRACE_REBUILD_OBJS)
```

`env` may not be in `/usr/bin`.
It would be more portable to let `/bin/sh` export them.

```suggestion
	$(Q) CFLAGS="$(CFLAGS) $(XCFLAGS) $(CPPFLAGS)" $(DTRACE) -G -C $(INCFLAGS) -s $(srcdir)/probes.d -o $@ $(DTRACE_REBUILD_OBJS)
```

---

That makes sense, I checked and that works so I'll commit it your way :+1: 

(I thought the location `of` `/usr/bin/env` was part of the POISX standard but it turns out it's not)